src/common/components/select-repository-list/index.js

```diff
@@ -55,18 +53,6 @@ export class SelectRepositoryListComponent extends Component {
 
   // TODO this can be moved too select-repository-row as the error prop is on
   // the repository object passed to it
-  getErrorMessage(error) {
-
-    const { payload = undefined } = error;
-
-    if (payload) {
-      if (payload.code === SNAP_NAME_NOT_REGISTERED_ERROR_CODE) {
-        return this.getSnapNotRegisteredMessage(payload.snap_name);
-      } else {
-        return payload.message;
-      }
-    }
-  }
```

Comment:
Some baggage left over from when the file was copied from `components/repositories-list/index.js`. The snap-name-not-registered error is never shown in this component, so this method is largely redundant.